.github/workflows/update-downstream.yml

```diff
@@ -25,10 +25,9 @@ jobs:
       REF_URL: https://github.com/${{ github.repository }}/tree/${{ github.sha }}
       SOURCE_REPO: ${{ github.repository }}
       SOURCE_PR_NUMBER: ${{ github.event.pull_request.number }}
-      SOURCE_PR_URL: https://github.com/${{ github.repository }}/pull/${{ github.event.pull_request.number }}
-      HEAD_REF: ${{ github.head_ref }}
-      HEAD_COMMIT_MSG: ${{ github.event.head_commit.message }}
-      DIFF_URL: https://github.com/${{ github.repository }}/compare/${{ github.head_ref }}..${{ github.sha }}
+      SOURCE_PR_URL: ${{ github.event.pull_request.html_url }}
+      BASE_REF: ${{ github.event.pull_request.base.ref }}
+      DIFF_URL: ${{ github.event.pull_request.diff_url }}
```

Comment:
This will only link to the actual diff file -- for this PR, it would look like: https://github.com/DeterminateSystems/detsys-ts/pull/53.diff

Reply:
Okay, I'll make sure we get both